native/packages/react-native-bpk-component-picker/src/BpkPickerItem.android.js

```diff
@@ -34,15 +34,24 @@ const styles = StyleSheet.create({
 
 const BpkPickerItem = props => {
   const { value, label, onPress, selected } = props;
+
+  const accessibilityTraits = ['button'];
+  if (selected) {
+    accessibilityTraits.push('selected');
+  }
+
   return (
-    <View style={styles.pickerItem}>
-      <BpkTouchableNativeFeedback
-        onPress={() => onPress && onPress(value)}
-        accessibilityLabel={label}
-      >
+    <BpkTouchableNativeFeedback
+      onPress={() => onPress && onPress(value)}
```

Comment:
This check for `onPress` isn't actually required. `onPress={onPress}` is good enough. It's a required prop anyways right?

Reply:
It's actually not required, because of the way the API works. Consumers use `BpkPickerItem` like this:

```
<BpkPicker>
    <BpkPickerItem label="foo" value="1" />
</BpkPicker>
```

And then `BpkPickerMenu` adds the `onPress` behaviour when the children are cloned and it's rendered.

Tbh this is probably a good indication that these shouldn't be the same component. I'll split it into two (internal one used for rendering, external one for consumers to pass data) in the refactor.